dependencies/pom.xml

```diff
@@ -39,6 +39,7 @@
         <version.lib.activemq>5.16.0</version.lib.activemq>
         <version.lib.animal-sniffer>1.18</version.lib.animal-sniffer>
         <version.lib.annotation-api>1.3.5</version.lib.annotation-api>
+        <version.lib.ant>1.10.13</version.lib.ant>
```

Comment:
Please manage this dependency in top level pom, as this is not for our customers to be used (this file is only for dependencies required at runtime)

Reply:
With `top level pom` do you mean in `helidon/microprofile/tests/tck/pom.xml` or  `helidon/pom.xml`?

Reply:
It was a dead code. I have removed it.

Initially I added it to execute the sigtest from a build.xml of tck-annotations, but it failed because the java ant task uses the security manager and that is not supported.